### ISSUE TITLE
Feature request: add dark mode to menu & dialog

### ISSUE THREAD

**Issue by lwchkg** [enz/pentobi#23]
Qt has a dark mode functionality. The following screenshot is a comparison of a dark mode enabled example (Qt Quick Controls) to Pentobi. I believe that Pentobi may be able to benefit from dark mode menu and dialogs that is already available in Qt.

If interested, we can track the discussion & progress in this issue. Anyway, I have no bandwidth to work on this by now. :-(

![image](https://user-images.githubusercontent.com/4113719/170065454-27172ade-b967-4e81-b23c-1de7b1a97659.png)

FYI, in the example dark mode is implemented by adding the config file below (named `qtquickcontrols2.conf`) to the compilation. Sadly this implementation appears unsuitable for Pentobi because users are allowed to switch light/dark themes. I am also unsure if prior use of custom controls will add to the difficulty of the implementation.
```
[Material]
Primary=#41cd52
Accent=#41cd52
Theme=System

[Universal]
Accent=#41cd52
Theme=System
```

**Comment by enz** [comment]
Yes, this would require some major changes because the Pentobi theme is unrelated to the QuickControls theme apart from the "System" Pentobi theme, which only works to some extent with the Fusion style (and only on KDE) because QuickControls item palette is not well implemented for all styles in Qt 5.

Generally, Pentobi is in maintenance mode and I wouldn't like to add any new features, apart from bug fixes or very small and safe changes.

**Comment by lwchkg** [comment]
I see. Just keep this on the back burner so someone may work with this later. Otherwise, I may have some time myself a few months later.

Also, thanks for pointing out about Qt5. I'll test on Qt5+Linux when I have the time to work on this. If this turns out to be complex, maybe its a good idea to implement only on Qt6 and Universal style.

**Comment by enz** [comment]
I'm planning to drop support for Qt5 anyway as soon as KDE makes the switch to Qt6 (otherwise Pentobi's KDE thumbnailer won't compile), so trying work around theming deficencies in Qt5 would be a waste of time.
